Python/symtable.c

```diff
@@ -1914,6 +1935,8 @@ symtable_visit_stmt(struct symtable *st, stmt_ty s)
         break;
     case ImportFrom_kind:
         VISIT_SEQ(st, alias, s->v.ImportFrom.names);
+        if (!check_import_from(st, s))
+            VISIT_QUIT(st, 0);
```

Comment:
```suggestion
        if (!check_import_from(st, s)) {
            VISIT_QUIT(st, 0);
        }
```

Not sure if PEP 7 applies here as "new code".